xdem/coreg/workflows.py

```diff
@@ -201,16 +205,28 @@ def dem_coregistration(
 
     if isinstance(ref_dem_path, str):
         if grid == "ref":
-            ref_dem, src_dem = gu.raster.load_multiple_rasters([ref_dem_path, src_dem_path], ref_grid=0)
+            ref_dem, src_dem = gu.raster.load_multiple_rasters([ref_dem_path, src_dem_path])
```

Comment:
Could you just tell me why you're removing the ref_grid?

Reply:
with a ref_grid in parameter, the function load_multiple_rasters perform the reprojection of one of the other DEM. If a shift is given, it has to be perform before any reprojection. That's why I removed the ref_grid so the DEM are just loaded, then the shift is applied to src_dem if applicable, and then I added the reprojection step.

Reply:
great thanks